### PR TITLE
Bugfix/cutter unit test

### PR DESCRIPTION
**Pull request by Shi26**

### CODE REVIEW

File: lexos/processors/prepare/cutter.py [hunk 1]
```diff
@@ -131,13 +99,13 @@ def strip_leading_lines(line_queue, num_lines):
     strip_leading_blank_lines(line_queue)
 
 
-def cut_by_characters(text: str, chunk_size: int, overlap: int,
-                      last_prop: float) -> List[str]:
+def cut_by_characters(text, chunk_size, overlap, last_prop):
```
Comment (chantisnake) [inline lexos/processors/prepare/cutter.py]: Type Hinting?

File: lexos/processors/prepare/cutter.py [hunk 2]
```diff
@@ -439,9 +407,10 @@ def cut_by_milestone(text: str, cutting_value: str) -> List[str]:
 
     Chunk boundaries should be created when every milestone appears
     :param text: the text to be chunked as a single string
-    :param cutting_value: # maybe we should change this variable name?
+    :param cutting_value:
```
Comment (chantisnake) [inline lexos/processors/prepare/cutter.py]: no comment

**Comment by Shi26** [comment]
the bug showed in the main cutter function probably has something to do with the bug in function: strip_leading_words

**Comment by chantisnake** [comment]
Comment out the test case and then push.

File: lexos/processors/prepare/cutter.py [hunk 3]
```diff
@@ -132,12 +101,13 @@ def strip_leading_lines(line_queue, num_lines):
...
 
 def cut_by_characters(text: str, chunk_size: int, overlap: int,
-                      last_prop: float) -> List[str]:
+                      last_prop: float):
     """Cuts the text into equally sized chunks.
 
-    The segment size is measured by counts of characters, with an option for an
-    amount of overlap between chunks and a minimum proportion threshold for the
-    last chunk.
+    where the segment size is measured
```
Comment (chantisnake) [inline lexos/processors/prepare/cutter.py]: format your documentation well, fix your new lines

**Pull request closed without merging**